src/lib/ydocker/changes_dialog.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
are you sure that it is safe to add it to files? I usually avoids this as it can lead to some nasty bugs.

Reply:
Ah, good point. I did think about that, then ran the module and clicked around and thought it was OK... so I included this.
But now I realize the `test/` directory has only a useless spec helper :scream: 
So I will omit this and use the usual way in yast repos (which I did not realize before)
```
Style/FrozenStringLiteralComment:
  Enabled: false
```